website/build_javadocs.sh

```diff
@@ -0,0 +1,181 @@
+
+    <groupId>org.apache.fluss</groupId>
+    <artifactId>fluss-javadoc-temp</artifactId>
+    <version>$VERSION</version>
+    <packaging>pom</packaging>
+
+    <properties>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
+    <modules>
+        <module>fluss-client</module>
+        <module>fluss-flink/fluss-flink-common</module>
+    </modules>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-javadoc-plugin</artifactId>
+                <version>3.6.3</version>
+                <configuration>
+                    <windowTitle>Apache Fluss \${project.version} API</windowTitle>
+                    <doctitle>Apache Fluss \${project.version} API</doctitle>
+                    <doclint>none</doclint>
+                    <failOnError>false</failOnError>
+                    <quiet>true</quiet>
+                    <additionalJOptions>
+                        <additionalJOption>--add-opens=java.base/java.lang=ALL-UNNAMED</additionalJOption>
+                        <additionalJOption>-J--add-exports=java.rmi/sun.rmi.registry=ALL-UNNAMED</additionalJOption>
+                    </additionalJOptions>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+</project>
+EOF
+
+# Cleanup function for temporary files
+cleanup() {
+  if [[ -f "$TEMP_POM" ]]; then
+    rm -f "$TEMP_POM"
+    echo "Cleaned up temporary POM file"
+  fi
+}
+
+# Set trap for cleanup on script exit
+trap cleanup EXIT
+
+# Set Maven options for Java 11+ compatibility
+export MAVEN_OPTS="--add-exports=java.rmi/sun.rmi.registry=ALL-UNNAMED --add-opens=java.base/java.lang=ALL-UNNAMED"
+
+echo "Building and generating Javadoc..."
+
+# Build and generate javadoc using temporary POM
+if (./mvnw -f "$TEMP_POM" clean compile javadoc:aggregate -q 2>/dev/null || mvn -f "$TEMP_POM" clean compile javadoc:aggregate -q); then
+  echo "Javadoc generation completed successfully"
+else
+  echo "Error: Javadoc generation failed"
+  exit 1
+fi
+
+# Copy generated javadoc to output directory
+if [[ -d "target/site/apidocs" ]]; then
+  cp -r target/site/apidocs/* "$OUTPUT_DIR/"
+  echo "Javadoc copied to: $OUTPUT_DIR"
+else
+  echo "Error: No javadoc files generated in target/site/apidocs"
+  exit 1
+fi
+
+# Create or update redirect page
+JAVADOC_INDEX="website/static/javadoc/index.html"
+mkdir -p "website/static/javadoc"
+cat > "$JAVADOC_INDEX" << EOF
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="refresh" content="0; url=${JAVADOC_VERSION}/">
+    <title>Apache Fluss API Documentation</title>
+</head>
+<body>
+    <p>Redirecting to <a href="${JAVADOC_VERSION}/">Apache Fluss ${JAVADOC_VERSION} API</a></p>
+</body>
+</html>
+EOF
```

Comment:
Do we really need this `index.html`?  As it is under `javadoc/index.html`, so multiple javadoc version will override this file.